.github/workflows/promote-to-stable.yml

```
@@ -0,0 +1,44 @@
+name: Promote to stable
+
+on:
+  issue_comment:
+    types:
+      - created
+
+permissions:
+  issues: write
+
+jobs:
+  promote-content:
+    name: ⬆️ Promote Content to stable
+    environment: "2204 Branch"
+    runs-on: ubuntu-latest
+    if: |
+      ( !github.event.issue.pull_request )
+      && contains(github.event.comment.body, '/promote ')
+      && contains(github.event.issue.body, '`ffmpeg-2204`')
```

```suggestion
      && contains(github.event.issue.body, '`ffmpeg-2204 `')
```

If you add a space here, you'll prevent this from triggering when the command is issued for the SDK snap :)